boards/posix/nrf_bsim/Kconfig.defconfig

```diff
@@ -44,7 +44,8 @@ config BT_CTLR
 	default y if BOARD_NRF52_BSIM || BOARD_NRF5340BSIM_NRF5340_CPUNET
 	depends on BT
 
-config HEAP_MEM_POOL_SIZE
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
```

Comment:
generally it's bad practice to define Kconfig settings in defconfig, the Kconfig.defconfig should only adjust existing defaults (which was the case when using `HEAP_MEM_POOL_SIZE`.

Comment applies to other updated Kconfig.defconfig as well.

Reply:
Sure, that's a good point. I'll do it in a follow-up PR.